sites/page-details/index.json-ld

```diff
@@ -163,10 +172,10 @@
 			},
 			"notes": {
 				"en": [
-					"Although the component has been updated to version 3.1, the version 2.0 and 3.0 are still supported. However, the page content and the &lt;footer&gt; have to be located inside an &lt;article&gt; tag."
+					"Although the component has been updated to version 3.2, versions 2.0, 3.0 and 3.1 are still supported. However, the page content and the &lt;section&gt; have to be located inside a &lt;main&gt; tag."
 				],
 				"fr": [
-					"Bien que le composant ait été mis à jour vers la version 3.1, la version 2.0 and 3.0 sont toujours pris en charge. Cependant, le contenu de la page et le &lt;footer&gt; doivent être situés à l'intérieur d'une balise &lt;article&gt;."
+					"Bien que le composant ait été mis à jour vers la version 3.2, les versions 2.0, 3.0 et 3.1 sont toujours prises en charge. Cependant, le contenu de la page et la &lt;section&gt; doivent être situés à l'intérieur d'une balise &lt;main&gt;."
```

Comment:
Revised the second sentence slightly since it seemed outdated/misleading (page details no longer uses ``<footer>`` and ``<article>`` elements have never been historically used instead of ``<main>``).